#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_warnings.dpatch by Mario Limonciello <superm1@ubuntu.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Clean up warnings during build

--- src/nautilus-image-resizer.c	2008-01-26 17:00:17.000000000 -0600
+++ src/nautilus-image-resizer.c	2008-02-02 04:01:17.000000000 -0600
@@ -164,7 +164,7 @@
 	NautilusImageResizerPrivate *priv = NAUTILUS_IMAGE_RESIZER_GET_PRIVATE (resizer);
 
 	GFile *parent_file, *new_file;
-	char *basename, *extension, *new_path, *new_basename;
+	char *basename, *extension, *new_basename;
 	
 	g_return_val_if_fail (G_IS_FILE (orig_file), NULL);
 
--- src/nautilus-image-rotator.c	2008-01-26 17:00:17.000000000 -0600
+++ src/nautilus-image-rotator.c	2008-02-02 04:02:13.000000000 -0600
@@ -161,7 +161,7 @@
 	NautilusImageRotatorPrivate *priv = NAUTILUS_IMAGE_ROTATOR_GET_PRIVATE (rotator);
 
 	GFile *parent_file, *new_file;
-	char *basename, *extension, *new_path, *new_basename;
+	char *basename, *extension, *new_basename;
 	
 	g_return_val_if_fail (G_IS_FILE (orig_file), NULL);
 
